lib/ui/semantics.dart

```diff
@@ -228,6 +228,9 @@ class SemanticsAction {
     _kSetTextIndex: setText,
   };
 
+  /// Temporary API until [values] return a list.
+  List<SemanticsAction> get valuesAsList => values.values.toList(growable: false);
```

Comment:
Mark this API as deprecated so no-one starts accidentally using it during the migration outside of the use case we have for it?

Reply:
Done, thanks.